viewer/src/services/queryService.js

```diff
@@ -0,0 +1,35 @@
+export const executeQuery = async (query, projectId, sourceName) => {
```

Comment:
Nice to break these into different api files describing what they do.  api/query and api/trace ..